src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -239,6 +239,12 @@ internal static partial class TextEditor
             //manual pinvoke for text-editor ScrollStateChangedSignal
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_ScrollStateChangedSignal")]
             public static extern global::System.IntPtr TextEditor_ScrollStateChangedSignal(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_Property_SELECTED_TEXT_START_get")]
+            public static extern int TextEditor_Property_SELECTED_TEXT_START_get();
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_Property_SELECTED_TEXT_END_get")]
```

Comment:
EntryPoint = "CSharp_TextEditor_Property_SELECTED_TEXT_END_get"

Reply:
@rabbitfor 
the function name is the same as in this patch
https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/dali-csharp-binder.git;a=blob;f=dali-csharp-binder/src/text-editor.cpp;h=2f30b82d354de9cb3aadd1e1465fa35f09d36876;hb=refs/heads/devel/master#l573